src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,12 +2,14 @@
 
 public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-	int totalRejected = 0;
+	private static final int REJECTED_AT_START = 0;

```

Comment:
I think that using a constant for zero initial value is an exaggeration (the name REJECTED_AT_START is clear from the declaration code).